samples/go/aws/lambda/template.yml

```diff
@@ -10,7 +10,7 @@ Resources:
           ASHERAH_KMS_KEY_ARN: arn:aws:kms:<region>:<account>:key/<id>
           ASHERAH_METASTORE_TABLE_NAME: EncryptionKey
       Handler: main
```

Comment:
Should this now be `Handler: bootstrap`?

Reply:
No, because the `func main()` is still the lambda entry point. According to docs only `The executable must be named bootstrap`